kong/plugins/correlation-id/schema.lua

```diff
@@ -9,8 +9,8 @@ return {
         type = "record",
         fields = {
           { header_name = { description = "The HTTP header name to use for the correlation ID.", type = "string", default = "Kong-Request-ID" }, },
-          { generator = { description = "The generator to use for the correlation ID. Accepted values are `uuid`, `uuid#counter`, and `tracker`. See [Generators](#generators).", type = "string", default = "uuid#counter",
-                          one_of = { "uuid", "uuid#counter", "tracker" }, }, },
+          { generator = { description = "The generator to use for the correlation ID. Accepted values are `uuid`, `uuid#counter`, and `tracker`. See [Generators](#generators).",
+                          type = "string", default = "uuid#counter", required = true, one_of = { "uuid", "uuid#counter", "tracker" }, }, },
```

Comment:
Just one question: if we make `the config.generator` field as a required parameter, for existing configurations in the database, without adding migrations, in hybrid mode, can the configuration sent down by CP pass DP's schema validation?

Reply:
I reckon DP can load successfully, let me add a test case to verify.